typescript/infra/scripts/helloworld/kathy.ts

```diff
@@ -370,71 +393,102 @@ async function sendMessage(
   const msg = 'Hello!';
   const expectedHandleGas = BigNumber.from(50_000);
 
-  const value = await retryAsync(
-    () =>
-      igp.quoteGasPaymentForDefaultIsmIgp(
-        origin,
-        destination,
-        expectedHandleGas,
-      ),
-    2,
-  );
+  // TODO sealevel igp support here
+  let value: string;
+  if (app.metadata(origin).protocol == ProtocolType.Ethereum) {
+    const valueBn = await retryAsync(
+      () =>
+        igp.quoteGasPaymentForDefaultIsmIgp(
+          origin,
+          destination,
+          expectedHandleGas,
+        ),
+      2,
+    );
+    value = valueBn.toString();
+  } else {
+    value = '0';
+  }
+
   const metricLabels = { origin, remote: destination };
 
   log('Sending message', {
     origin,
     destination,
-    interchainGasPayment: value.toString(),
+    interchainGasPayment: value,
   });
 
+  const sendAndConfirmMsg = async () => {
+    const sender = getAddressesForKey(keys, origin, multiProvider);
+    const tx = await app.populateHelloWorldTx(
+      origin,
+      destination,
+      msg,
+      value,
+      sender,
+    );
+
+    let txReceipt: TypedTransactionReceipt;
+    if (tx.type == ProviderType.EthersV5) {
+      // Utilize the legacy evm-specific multiprovider utils to send the transaction
+      const receipt = await multiProvider.sendTransaction(
+        origin,
+        tx.transaction,
+      );
+      txReceipt = {
+        type: ProviderType.EthersV5,
+        receipt,
+      };
+    } else if (tx.type === ProviderType.SolanaWeb3) {
+      // Utilize the new multi-protocol provider for non-evm chains
+      // This could be done for EVM too but the legacy MP has tx formatting utils
+      // that have not yet been ported over
+      const connection = app.multiProvider.getSolanaWeb3Provider(origin);
+      const payer = Keypair.fromSecretKey(base58ToBuffer(keys[origin]));
```

Comment:
ditto

Reply:
Actually this is intended to stay even after getCoreConfigStub is removed. Is there an alternative way you'd prefer we do the sealevel signing?